src/client/src/components/Item/Item.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Checkbox } from "../Checkbox/Checkbox";
+import { Label } from "../Label/Label";
+import { Image } from "../Image/Image";
+import delete_icon from "../../images/delete_icon.svg";
+import "./item.css";
+
+export const Item = ({ item, handleItemDelete, HandleItemUpdate }) => {
+  return (
+    <li className="list-item">
```

Comment:
better to destrut the properties you need from item at top of ifle that way you want have to keep doing item.something everywhere